Cargo.toml

```diff
@@ -24,11 +24,10 @@ exclude = [".github/"]
 codegen-units = 1
 lto = true
 
-[profile.test-release]
-inherits = "release"
-debug = true
-debug-assertions = true
-overflow-checks = true
+[profile.test-dev]
+inherits = "dev"
+opt-level = 1
+codegen-units = 16
```

Comment:
I believe this is the default value? Any particular reason to fix it?

Reply:
The default is `256` (https://doc.rust-lang.org/cargo/reference/profiles.html#dev).

I tried with that default and build and execution times are pretty much the same, so let me remove it actually.

Reply:
`256` is the default for incremental builds, which shouldn't be used in CI because the rust cache disables it iirc. So the "default" is `16`.

Reply:
I see, then maybe that's why the difference is non-existent :smile: